src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -3,6 +3,8 @@ import React, { FunctionComponent } from "react";
 
```

Comment:
will we be moving this to tt-components? or the new redesign would be different

Reply:
i think not, will leave it at creator, since only creator UI have progressBars.